Leafly-com/lm-inventory-service
# LMDISC-82_Decrement-Inventory

# Description

Adds endpoint `POST /v1/inventory/decrement` that decrements `availableQuantity` from an associated list of skus.

## Manual Testing

`npm run seed:local`
`npm run start`
`POST http://localhost:8080/v1/inventory/decrement`
```
{
  "inventoryDecrement": [
    {
      "sku": "1",
      "decrement": 1
    },
    {
      "sku": "2",
      "decrement": 2
    }
  ]
}
```

Confirm the quantities have been updated after performing the operation

![image](https://user-images.githubusercontent.com/54331819/65998548-1f52ee00-e461-11e9-81d4-5dafc36adbfd.png)

## Unit/e2e Testing

`npm run test`

![image](https://user-images.githubusercontent.com/54331819/65998636-575a3100-e461-11e9-9e0e-e707cb6c8525.png)
